### PR TITLE
added some feature , and solve the problem which i have with account

### PR DESCRIPTION
**Pull request by SauravDharwadkar**
edit this 
and check readme for more info bro

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,6 +32,14 @@ If true, adds a random delay (10s-30s) before joining a meeting. Can be useful s
 - **auto_leave_after_min:**  
 If set to a value greater than zero, the bot leaves every meeting after the specified time (in minutes). Useful if you know the length of your meeting, if this is left a the default the bot will stay in the meeting until a new one is available.
 
+- **additional_acc:** 
+if u have addititonal account in same mail or anything similar to given pic 
+<img width="80%" alt="jarray reverse exampl" src="https://i.imgur.com/VPmecz2.png">
+change additional_acc by which no of account u want to select "counting start from 1 here"
```
Comment (TobiasPankner) [inline README.md:38]: What you mention here as accounts are actually organisations, this has to be changed everywehere.
```suggestion
- **organisation_num:** 
If your Teams account is in multiple organisations, as seen in the example below, change the organisation_num to the number of the list item (counting starts from 1) 
<img width="80%" alt="jarray reverse exampl" src="https://i.imgur.com/VPmecz2.png">
```

File: README.md [hunk 2]
```diff
@@ -32,6 +32,14 @@ If true, adds a random delay (10s-30s) before joining a meeting. Can be useful s
...
+
+- **headless:**
+if ypu want to run code without opening instance of chrome change value to true
```
Comment (TobiasPankner) [inline README.md:41]: Typo and reformatting the sentence to better match previous ones.
```suggestion
If true, runs Chrome in headless mode (does not open GUI window and runs in background).
```

File: auto_joiner.py [hunk 3]
```diff
@@ -169,6 +169,15 @@ def wait_until_found(sel, timeout):
...
 
+def wait_until_found_xpath(sel, timeout):
+    try:
+        element_present = EC.visibility_of_element_located((By.XPATH, sel))
+        WebDriverWait(browser, timeout).until(element_present)
+
+        return browser.find_element_by_xpath(sel)
+    except exceptions.TimeoutException:
+        print("Timeout waiting for element.")
+        return None
```
Comment (TobiasPankner) [inline auto_joiner.py:180]: This is just not needed, please delete all of this and use the wait_until_found function

File: auto_joiner.py [hunk 4]
```diff
@@ -262,14 +271,18 @@ def hangup():
...
     chrome_options.add_argument('--ignore-certificate-errors')
     chrome_options.add_argument('--ignore-ssl-errors')
     chrome_options.add_argument("--use-fake-ui-for-media-stream")
     chrome_options.add_experimental_option('excludeSwitches', ['enable-logging'])
-    browser = webdriver.Chrome(ChromeDriverManager().install(), options=chrome_options)
 
-    load_config()
+    if config['headless'] :
+        chrome_options.add_argument('--headless')
+        print("headless mode enable..")
```
Comment (TobiasPankner) [inline auto_joiner.py:284]: Some nicer phrasing
```suggestion
        print("Enabled headless mode")
```

File: auto_joiner.py [hunk 5]
```diff
@@ -283,7 +296,7 @@ def main():
         login_email = wait_until_found("input[type='email']", 5)
         if login_email is not None:
             login_email.send_keys(Keys.ENTER)
-
+            print("\nWorking...")
```
Comment (TobiasPankner) [inline auto_joiner.py:299]: I don't quite understand why this is here, please delete this line

File: auto_joiner.py [hunk 6]
```diff
@@ -304,6 +317,17 @@ def main():
...
+    if config['additional_acc']  > 1 :
+        select_change_acc=wait_until_found_xpath("/html/body/div[1]/div[1]/app-header-bar/div/div/button[1]",20)
+        if select_change_acc is not None :
+            select_change_acc.click()
+            time.sleep(1)
+            change_acc = wait_until_found_xpath(f"/html/body/div[5]/div/ul/settings-dropdown-tenants/li[{config['additional_acc']}]/button[1]",20)
+            # /html/body/div[5]/div/ul/settings-dropdown-tenants/li[2] also usable
+            if change_acc is not None :
+                change_acc.click()
+
```
Comment (TobiasPankner) [inline auto_joiner.py:330]: Change from xpath to css selector so it is the same as in the other parts of the code, change phrasing and make more error proof
```suggestion
    # if additional organisations are setup in the config file
    if config['organisation_num']  > 1 :
        additional_org_num = config['organisation_num']
        select_change_org=wait_until_found("button.tenant-switcher",20)
        if select_change_org is not None :
            select_change_org.click()
            
            change_org = wait_until_found(f"li.tenant-option[aria-posinset='{additional_org_num}']",20)
            if change_org is not None:
                change_org.click()
    
```

File: config.json.example [hunk 7]
```diff
@@ -4,6 +4,8 @@
   "start_automatically": false,
   "random_delay": false,
   "auto_leave_after_min": -1,
+  "additional_acc" : 1,
```
Comment (TobiasPankner) [inline config.json.example:7]:
```suggestion
  "organisation_num" : 1,
```

**Comment by SauravDharwadkar** [comment]
Nice !

**Comment by TobiasPankner** [comment]
Make the changes I commented and make another pull request

**Comment by SauravDharwadkar** [comment]
Ok

**Pull request closed without merging**